pkg/ddevapp/laravel.go

```diff
@@ -56,6 +56,7 @@ func laravelPostStartAction(app *DdevApp) error {
 		"DB_USERNAME":   "db",
 		"DB_PASSWORD":   "db",
 		"DB_CONNECTION": dbConnection,
+		"MAIL_HOST":     "127.0.0.1",
```

Comment:
Doesn't it need the port as well, 1025?

Reply:
The port is set to `1025` by default.

Composer copies over the `.env.example` which has the following mail related settings:

```env
MAIL_MAILER=smtp
MAIL_HOST=mailhog
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="hello@example.com"
MAIL_FROM_NAME="${APP_NAME}"
```

You can see that the default mail host doesn't work OOTB.

Reply:
Update PR to include port if you would prefer it in

Reply:
1025 is an unusual default mail port (25 is normal). Oh, I see, it's already there `MAIL_PORT=1025`. But we didn't have `MAIL_HOST` set right. Thanks.